src/settings/handlers/AccountSettingsHandler.js

```diff
@@ -73,6 +73,10 @@ export default class AccountSettingsHandler extends MatrixClientBackedSettingsHa
             return !content['disable'];
         }
 
+        if (settingName === "notificationsEnabled") {
+            return this._getSettings("m.notification.sound");
```

Comment:
With my spec hat on: please don't use the m.* namespace without a proposal post-FCP. The rationale is that anything experimental should be in a custom namespace (either `im.vector` or `uk.half-shot` or something) to avoid potential clashes with future Matrix things. Riot in particular is a great test bed for these kinds of things because the settings system is capable of handling a migration period for if/when a proposal is approved and things move to m.* namespaces.